app/Http/ContentSecurityPolicy.php

```diff
@@ -56,15 +56,16 @@ public function configure(): void
             Keyword::SELF,
             Keyword::UNSAFE_INLINE,
             Keyword::REPORT_SAMPLE,
+            'https://unpkg.com',
```

Comment:
Maybe only add this when not in prod? Not sure how much it matters.

Reply:
I was thinking about stripping it out at build time. Might circle back and do that later.

Reply:
After further review, this is indeed used for some Mailbook features, though we're not using them right now. In the interest of keeping things simple I'll leave this in the CSP.